services/ui-src/src/verbiage/pages/accordion.ts

```diff
@@ -82,8 +106,8 @@ export default {
         },
         {
           type: "text",
-          as: "span",
-          content: "<br><br><b>Admin instructions</b>",
+          as: "p",
+          content: "<b>Admin instructions</b>",
```

Comment:
Seems like there's inconsistent usage of `<b>` vs `<strong>`. Most of these probably should be `<strong>` to indicate importance versus it being bolded just for style. But I'd leave it as-is for now..

Reply:
Good point! Are they handled differently from an html or accessibility perspective? I guess I never considered why you might pick one over the other.

Reply:
In theory, screen readers can read those tags in different voices, but the tag differences are more to convey meaning. `<strong>` indicates "I want you to pay particular attention to this word/phrase" while `<b>` is more "I think this makes this word/phrase look better."

Reply:
💡good to know!